pyproject.toml

```diff
@@ -8,6 +8,7 @@ license = "MIT"
 
 [tool.poetry.dependencies]
 python = "~3.11"
+pillow = "^10.0.0"
```

Comment:
This should be locked to the bugfix version just to be sure (`~` instead of `^`)

Reply:
Fixed. Didn't know the difference, and I think poetry did that automatically.